src/Control/Director.php

```diff
@@ -76,40 +80,47 @@ class Director implements TemplateGlobalProvider
     private static $alternate_base_folder;
```

Comment:
I think we should remove alternate_base_folder in favour of alternate_base_url.

Note that this doesn't need to be logged as an API change as it was introduced in 4.